eWonConnector-gateway/src/main/java/org/imdc/ewon/SyncManager.java

```diff
@@ -698,6 +713,37 @@ protected void updateTagValues(EwonData data) {
             BasicScanclassHistorySet historySet =
                     new BasicScanclassHistorySet(providerName, "_exempt_", -1);
 
+            // Add AllRealtime tag under Ewon _config tag folder
+            if (!registeredEwons.contains(device))
+            {
+                // Create path to Ewon's all realtime tag
+                String ewonRealtimePath = buildTagPath(device + "/_config/AllRealtime");
+
+                // Add Ewon to registered Ewon list
+                registeredEwons.add(device);
+
+                // Register all realtime tag write handler with provider
+                provider.registerWriteHandler(ewonRealtimePath, new WriteHandler() {
+                    public QualityCode write(TagPath tagPath, Object o) {
+                        try {
+                            boolean rt = (boolean) o;
```

Comment:
Consider a more descriptive name for rt.

Reply:
changed